### PR TITLE
SWC-7063

### PR DESCRIPTION
**Pull request by jay-hodgson**

### CODE REVIEW

**Comment by jay-hodgson** [comment]
Screenshot 2024-11-07 at 10 38 25 AM

Screenshot 2024-11-07 at 10 57 24 AM

**Comment by jay-hodgson** [comment]
Design: 
Screenshot 2024-11-07 at 11 00 58 AM

**Comment by jay-hodgson** [comment]
(Note, the help icon size and position has been fixed since the screenshot was taken)

File: src/main/java/org/sagebionetworks/web/client/widget/entity/EntityMetadata.java [hunk 1]
```diff
@@ -133,22 +124,14 @@ public void configure(
...
         versionHistoryWidget.setVisible(false);
       }
-      view.setRestrictionPanelVisible(
-        en instanceof TableEntity ||
-        en instanceof Folder ||
-        en instanceof DockerRepository
-      );
-    }
-    if (bundle.getEntity() instanceof Folder) {
-      containerItemCountWidget.configure(bundle.getEntity().getId());
```
Comment (nickgros) [inline src/main/java/org/sagebionetworks/web/client/widget/entity/EntityMetadata.java:148]: Thanks for removing this unused code!

**Review by nickgros — APPROVED** [review]
LGTM

**Pull request merged**